inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,86 +1,412 @@
+
+** For more installation details, see the https://inception-project.github.io/documentation/[Admin Guide].
+
+
+==== Download and Import an Example Project
+
+In order to understand what you read in this guide, it makes sense to have an annotation project to look at and click through. We created some example projects. You find them in the section https://inception-project.github.io/example-projects/[Example Projects] on our website.
+
+image:getting_started_download_example_project.png[image,width=636,height=128]
+
+
+*Step 1 - Download:* For this guide, we use the _Interactive Concept Linking_ project. Please download it from the https://inception-project.github.io/example-projects/[Example Projects] section on our website. It constists of two documents about pets. The first one contains some annotations as an example, the second one is meant to be your playground. It has originally been created for concept linking annotation but in every project, you can create any kinds of annotations.
+
+NOTE: *Concept Linking:* This is the kind of annotation telling whether the annotated text part refers to a
+Person (in {product-name}, it will per default get the tag PER), Organization (ORG), Location (LOC) or any other (OTH). +
+The respective layer to annotate this aspect is the _Named Entity_ layer. If you are not sure what layers are, check the box on xref:layers_and_features_in_getting_started[_Layers And Features_] in the section <<Project Settings>>. 
+
+* *Step 2 - Import:* After logging into
+{product-name}, click on the blue folder on the top right (next to „Import project archives“) and browse for the example project you have
+downloaded in Step 1. Finally, click the grey upload button right next to
+the blue folder to upload the selected project. The project has now been added and you can use it to follow the explanations of the next section.
+
+
+== Settings and Structure Of an Annotation Project
+In this section, we will have a look on the structure of every annotation project in {product-name}. If you prefer to make some annotations first, you may go on with <<First Annotations with {product-name}>> and return later. This section here gives you a basic orientation in {product-name}.
+
+You may have different projects in {product-name} at the same time. First, let‘s see what elements each project _has_ and where you can adjust these elements by examining the <<Project Settings>>. Second, we will find out what you can _do_ in each project having a look on the <<Structure Of an Annotation Project>>.
+
+
+=== Project Settings
+
+Each project consists of the following elements:
+
+* one or (usually) more *Documents* to annotate
+* one or (usually) more *Users* to work on the project
+* one or (usually) more *Layers* to annotate with
+* Optional: one or more *Knowledge Base/s*
+* Optional: *Recomenders* to automatically suggest annotations
+* Optional: *Guidelines* for you and your team
```

Comment:
It's not good to refer to them later since we go through the project settings now. But it's important to say that the mentioned ones are not all, that's right. I got that right. :+1: